includes/Modules/Analytics_4.php

```diff
@@ -503,7 +503,7 @@ function( $property ) {
 	protected function setup_info() {
 		return array(
 			'slug'        => self::MODULE_SLUG,
-			'name'        => _x( 'Analytics 4 (Alpha)', 'Service name', 'google-site-kit' ),
```

Comment:
This wasn't called for in the ACs but might make sense to change at the same time.